project/about/assets/css/team.scss

```diff
@@ -0,0 +1,206 @@
+article.that{
+    .page-title{
+      text-align: center;
+      //text-decoration: underline;
```

Comment:
Remove commented code that is not needed (or is not going to be needed)

Reply:
ase mas kouklitsa m